lib/TTT/minimax.ex

```diff
@@ -36,13 +40,21 @@ defmodule Minimax do
 
     case current do
       ^max ->
-        Enum.reduce(moves, %{player: nil, position: nil, score: -10}, fn move, acc ->
-          if move.score >= acc.score, do: move, else: acc
+        Enum.reduce(moves, %{position: nil, score: -10}, fn move, acc ->
+          if move.score > acc.score do
+            move
+          else
+            acc
+          end
         end)
 
       ^min ->
-        Enum.reduce(moves, %{player: nil, position: nil, score: 10}, fn move, acc ->
-          if move.score <= acc.score, do: move, else: acc
+        Enum.reduce(moves, %{position: nil, score: 10}, fn move, acc ->
```

Comment:
Interesting to me that this namespace is using `20`/`-20` _and_ `10`/`-10`. Are these [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))? I'm a little bit confused about how much I could change these numbers before tests would start failing.